### PR TITLE
Converting Flow model to hibernate

### PR DESCRIPTION
**Pull request by ShridharSattur**
Converting Flow model to hibernate

### CODE REVIEW

File: wherehows-dao/src/main/java/wherehows/models/Flow.java [hunk 1]
```diff
@@ -13,16 +13,47 @@
...
+    @Transient
+    private String modified;
+
+    @Transient
+    private String appCode;
+
+    @Column(name = "flow_group")
+    private String group;
+
+    @Column(name = "app_id")
```
Comment (mars-lan) [inline wherehows-dao/src/main/java/wherehows/models/Flow.java]: This table has composite primary key (app_id, flow_id)

File: wherehows-dao/src/main/java/wherehows/models/Flow.java [hunk 2]
```diff
@@ -13,16 +13,47 @@
...
-    public Integer level;
-    public int jobCount;
-    public String created;
-    public String modified;
-    public String appCode;
-    public String group;
-    public Integer appId;
+    @Id
+    @Column(name = "flow_id")
+    private Long id;
```
Comment (mars-lan) [inline wherehows-dao/src/main/java/wherehows/models/Flow.java]: Let's rename this to flow_id to be clear

**Comment by mars-lan** [comment]
Here's the style file for IDEA: https://github.com/linkedin/WhereHows/blob/master/wherehows-docs/LinkedIn%20Style.xml

File: wherehows-dao/src/main/java/wherehows/models/Flow.java (hunk 2 above)
Comment (ShridharSattur) [inline wherehows-dao/src/main/java/wherehows/models/Flow.java]: Updated it

File: wherehows-dao/src/main/java/wherehows/models/Flow.java (hunk 1 above)
Comment (ShridharSattur) [inline wherehows-dao/src/main/java/wherehows/models/Flow.java]: Handled it now

File: wherehows-frontend/app/dao/FlowRowMapper.java [hunk 3]
```diff
@@ -19,40 +19,39 @@
...
-        Long id = rs.getLong(FLOW_ID_COLUMN);
-        String name = rs.getString(FLOW_NAME_COLUMN);
-        Integer appId = rs.getInt(APP_ID_COLUMN);
-        String appCode = rs.getString(APP_CODE_COLUMN);
-        String group = rs.getString(FLOW_GROUP_COLUMN);
-        String path = rs.getString(FLOW_PATH_COLUMN);
-        Integer level = rs.getInt(FLOW_LEVEL_COLUMN);
+public class FlowRowMapper implements RowMapper<Flow> {
+  public static String FLOW_ID_COLUMN = "flow_id";
+  public static String JOB_ID_COLUMN = "job_id";
```
Comment (mars-lan) [inline wherehows-frontend/app/dao/FlowRowMapper.java]: Are these actually used outside of this class? If not, let's make them private

File: wherehows-dao/src/main/java/wherehows/models/Flow.java [hunk 4]
```diff
@@ -13,16 +13,56 @@
...
+  private String appCode;
+
+  @Column(name = "flow_group")
+  private String group;
+
+  @Id
+  @Column(name = "app_id")
+  private Integer appId;
+
+  static class FlowKeys {
```
Comment (mars-lan) [inline wherehows-dao/src/main/java/wherehows/models/Flow.java]: Can we make this private too?

File: wherehows-frontend/app/dao/FlowRowMapper.java (hunk 3 above)
Comment (ShridharSattur) [inline wherehows-frontend/app/dao/FlowRowMapper.java]: These are used in FlowsDAO and SearchDAO.

**Pull request merged**